src/software/firmware/includes/parameters.h

```diff
@@ -79,25 +79,41 @@ static const int32_t PID_PATIENT_INTEGRAL_MIN = -1000;
 #define VALVE_CLOSED_STATE 105
 
 #define SERVO_VALVE_PERIOD 20000
-#define PIN_SERVO_BLOWER D2  // PA10 / TIM1_CH3
 #define TIM_CHANNEL_SERVO_VALVE_BLOWER 3
-#define PIN_SERVO_PATIENT D4  // PB5 / TIM3_CH2
 #define TIM_CHANNEL_SERVO_VALVE_PATIENT 2
 
+#if HARDWARE_VERSION == 1
+#define PIN_SERVO_BLOWER D2   // PA10 / TIM1_CH3
```

Comment:
Please check initialisation of servoBlower and escBlower. If we update PINs, we also need to use the proper hardware timer and channel.
I use this page to know which timer and channel use for servo and esc. https://os.mbed.com/platforms/ST-Nucleo-f411re/

Hello @ABOSTM can you verify if there is another problem with hardwareTimers and channel? I don't want to miss something.

Reply:
As we are not in Mbed, but in arduino, it would be more reliable to look at https://github.com/stm32duino/Arduino_Core_STM32/blob/master/variants/NUCLEO_F4x1RE/PeripheralPins.c
Sometimes, several timers can be used by a single pin (ex: PA2 can be used with TIM2 or TIM5), the file I provided give you the default choice which has been made in Arduino.
Note: the default choice can be overwritten bey redefining the whole array in you code.